app/src/main/res/values/style.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="edit_text_style" >
```

Comment:
this it not the correct name style defined for Mobile Tandil, pls review the code style

Reply:
done

Reply:
The styles need to be in Camel Case and starting with capital letter. pls rename all the styles

Reply:
done